README.md

```diff
@@ -216,7 +288,7 @@ A `ThirdPartyNotices.txt` file is provided in the extension's source code listin
 - The first time you install the extension, you'll need to execute the `run` command at least once in order to access auto-completion.
 - While running a code file, if you get an error saying it can't find the file, make sure you've clicked on a valid Python code file before running it.

```

Comment:
the capitalization of "Python" seems inconsistent (?)

Reply:
good catch; fixed